### DISCUSSION TITLE
Is there a way to toggle animation like in the v1.

### DISCUSSION THREAD

**Discussion by sten-henriksson** [flyingpie/windows-terminal-quake#149] [Q&A] [answered]
Tried to use "ToggleDurationMs": 5 but it do not work

**Comment by flyingpie** [comment, marked as answer]
@sten-henriksson Hey Sten! The documentation and configuration are at the moment quite obscure, sorry about that!

This is an active work in progress, in the mean time, here's the setting:

```jsonc
{
  "AnimationDurationMs": 200
}
```

**Reply by sten-henriksson** [reply]
the v2 is so awsome btw absolutely love it. How stable is it on plasma

**Reply by flyingpie** [reply]
Thank you so much! Comments like yours have made it to what it is, and what it will be in the future :)

I'd call the Plasma build pretty stable, and close to being on feature-parity with Windows.

The main issue at the moment though, is that it doesn't have a super nice way of starting it. It's basically a command line app, and doesn't have a tray icon in 2.0.11 yet.

My goal is to publish Flatpaks, and hopefully put it on Flathub as well.
The next release will also include a tray icon on Plasma, and a GUI.
